modules/reference/pages/rpk/index.adoc

```diff
@@ -10,3 +10,5 @@ You can also get descriptions for each flag by running `rpk <command> --help` in
 All `rpk` commands feature autocompletion. To use the feature, press tab. 
 
 For more information on autocompletion, see xref:./rpk-generate/rpk-generate-shell-completion.adoc[`rpk generate shell-completion`]. 
+
+For more information on deprecated `rpk` commands and flags, see xref:upgrade:deprecated/index.adoc#rpk-commands[Deprecated features - rpk commands].
```

Comment:
```suggestion
For more information on deprecated `rpk` commands and flags, see xref:upgrade:deprecated/index.adoc[Deprecated features - rpk commands].
```
I was initially confused with the direct link into the rpk commands section, so suggest linking to the top of the page.

Reply:
Also, there's a typo in the description for `rpk registry` on this index page that you could clean up in this PR: 
_These commands let you manage `rpk`registry._ 
I think it just needs a space so the `rpk` renders properly. And there's also an instance of rpk in the intro that should be `rpk`.

Reply:
And I think the intro is a little hard to read, with so many short sentences each in a separate paragraph. Maybe if you put this line into a note, it'll break up the intro content and look easier to digest: 
All `rpk` commands feature autocompletion. To use the feature, press tab.

Reply:
Yeah I hated how this intro came to be. We kept adding more stuff into it and now it's not good